pkg/ccl/backupccl/show.go

```diff
@@ -448,7 +449,15 @@ you must pass the 'encryption_info_dir' parameter that points to the directory o
 						len(dest)))
 			}
 		}
-		return infoReader.showBackup(ctx, &mem, mkStore, info, p.User(), resultsCh)
+		if err := infoReader.showBackup(ctx, &mem, mkStore, info, p.User(), resultsCh); err != nil {
+			return err
+		}
+		if backup.InCollection == nil {
+			telemetry.Count("show-backup.deprecated_syntax")
```

Comment:
ditto on adding more description to the metric.

Reply:
Done.